project-codeflare/codeflare-sdk
# Removal of init_myservice initContainer

# Issue link
[Jira Issue](https://issues.redhat.com/browse/RHOAIENG-19)
Closes #297 

# What changes have been made
Removal of the init_myservice initContainer for the ray workgroupspec because the operator also injects a wait-gcs-ready initContainer which does similar thing.

# Verification steps
Install codeflare sdk on a cluster - test notebooks and ensure that they work correctly.
Run unit tests and ensure that they work correctly.

## Checks
- Testing Strategy
